openedx/core/djangoapps/discussions/models.py

```diff
@@ -55,85 +55,158 @@ class Features(Enum):
     WCAG_2_1 = 'wcag-2.1'
     WCAG_2_0_SUPPORT = 'wcag-2.0-support'
 
+class LINKS(Enum):
+    """
+       Links to be used in discussion providers for documentation
+    """
+    PIAZZA_DOCUMENTATION: ''
+    PIAZZA_ACCESSIBILITY_DOCUMENTATION: ''
+    PIAZZA_EMAIL: ''
+    PIAZZA_CONFIGURATION_DOCUMENTATION: ''
+    PIAZZA_LEARN_MORE: ''
+
```

Comment:
@saadyousafarbi should I continue with enums or go for the hardcoded links

Reply:
i dont think we need `enums` here. We can hardcode.